var/spack/repos/builtin/packages/jdk/package.py

```diff
@@ -22,60 +22,167 @@
 # License along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 ##############################################################################
-#
-# Author: Justin Too <too1@llnl.gov>
-#
-import distutils.dir_util
+from spack.util.prefix import Prefix
 from spack import *
 
+import llnl.util.tty as tty
+import distutils.dir_util
+import glob
+import os
+
 
 class Jdk(Package):
-    """The Java Development Kit (JDK) released by Oracle Corporation
-       in the form of a binary product aimed at Java developers."""
+    """The Java Development Kit (JDK) released by Oracle Corporation in the
+    form of a binary product aimed at Java developers. Includes a complete JRE
+    plus tools for developing, debugging, and monitoring Java applications."""
+
     homepage = "http://www.oracle.com/technetwork/java/javase/downloads/index.html"
 
+    maintainers = ['justintoo']
```

Comment:
@justintoo I noticed you listed yourself as an author of this package above. Spack now has the ability to list `maintainers` of a package. Would you like to be added so that you are pinged whenever a major change to the package is made?